src/node-backend/models/domain-objects.js

```diff
@@ -10,21 +10,25 @@ export function JoiningStudent(fullName, email = "No email set") {
     this.joinCode = false;
 }
 
-export const Status = {
-    Created: "Created",
-    Assigned: "Assigned",
-    InProgress: "In Progress",
-    NeedsHelp: "Needs Help",
-    StudentDone: "Done",
-    Grading: "Grading",
-    Finished: "Finished"
-};
+export const Status = Object.freeze({

```

Comment:
why freeze?

Reply:
I saw that it's good practice to freeze if we don't want the object to be changed (e.g. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze), do you agree?